README.md

```diff
@@ -60,7 +60,76 @@ When creating a new entity definition you may use the following files as a guide
 
 For more concrete examples, you can take a look at the files located on the [definitions](./definitions) folder. 
 
-#### Conditions
+#### Synthesis
+
+The definition file may contain a synthesis section which is optional *only* if you can ensure that the telemetry is being stamped with the entity GUID and tags based on rules defined internally at NewRelic.
```

Comment:
This sentence reads a bit confusing.  I would just describe what the synthesis section does and maybe just mention that some types (e.g. all INFRA- or APP-) do not allow them for historical reasons.